XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -133,28 +133,14 @@ public static void DownloadAndInstallNewClient(ClientUpdateAlert updateAlert, IW
                 break;

```

Comment:
`using System.Diagnostics;` isn't used (GH doesn't let me comment on that line 🤦 )